torchinductor/codecache.py

```diff
@@ -192,6 +193,9 @@ def load(cls, source_code):
 
 
 class AsyncCompile:
+    def __init__(self):
+        self._context_keepalive = None
```

Comment:
In C++, I would recommend RAII and making a clear object whose job it is to maintain a 1:1 lifecycle with the cuda context.

However, looking at the docs, it seems like this api is undocumented and also not called by us directly. So I guess this is the best we can do.